### PR TITLE
Refactor CI

### PR DESCRIPTION
**Pull request by sdkrystian**

### CODE REVIEW

File: include/boost/json/value.hpp [hunk 1]
```diff
@@ -2479,12 +2480,13 @@ class value
             ! std::is_same<T, bool>::value,
                 T>::type
     {
+        using limits = std::numeric_limits<T>;
```
Comment (vinniefalco) [inline include/boost/json/value.hpp:2483]: Is the alias necessary for fixing the warnings?

Comment (vinniefalco) [reply include/boost/json/value.hpp:2483]: If you are going to change it use `DBL_MIN` and `DBL_MAX` or whatever those macros are called

Comment (sdkrystian) [reply include/boost/json/value.hpp:2483]: The alias isn't necessary, I just added it for convenience. We can use the macros instead

**Comment by vinniefalco** [comment]
Merged, thanks

**Pull request closed without merging**